src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx

```diff
@@ -9,7 +9,8 @@ import {
 } from '~app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.styles';
 
 const DkgOperator = ({ operator }: { operator: IOperator }) => {
-    const classes = useStyles({ operatorLogo: operator.logo, dkgEnabled: !checkDkgAddress(operator.dkg_address ?? '') });
+    const dkgEnabled = !checkDkgAddress(operator.dkg_address ?? '');
```

Comment:
Can we rename `checkDkgAddress` to `validateDKGAddress` ?
And have this functions somewhere in /validators folder in dkg.ts file. 
We have many different validators and it's good to have more robust structure for the app

Reply:
changed to validateDkgAddress, and we have metadataHelper file which contains all staff which related to metadata include this function